README.adoc

```diff
@@ -118,12 +79,55 @@ java -jar target/optaweb-vehicle-routing-backend-*.jar \
 [CAUTION]
 .Using large OSM files
 ====
-For best user experience it is recommended to use smaller regions like individual European or US states.
+For best user experience, use smaller regions like individual European or US states.
 Using OSM files larger than 1 GB will require significant RAM size
 and take a lot of time (up to several hours) for the initial processing.
 ====
 
-== Creating Custom Data Sets
+
+=== Run Frontend
+
+. Install `npm`. If you're on Fedora, you can do this by running:
++
+[source,shell]
+----
+sudo dnf install npm
+----
+
+. Install dependencies:
++
+[source,shell]
+----
+cd optaweb-vehicle-routing-frontend
+npm install
+----
+
+. Run frontend:
++
+[source,shell]
+----
+npm start
+----
+
+. Open http://localhost:3000/ . By default, the `npm start` command attempts to open this URL in your default browser.
+
+[TIP]
+.Prevent `npm start` from launching your default browser
+====
+If you don't want `npm start` to open a new browser tab each time you run it,
+export an environment variable `BROWSER=none`.
+
+You can use `.env.local` file to make this preference permanent.
+To do that, run
```

Comment:
Again, things like this, "To do that, run", are throughout. Very casual, and stylistically incorrect. I'll stop at this point and assume this is not intended to be Red Hat enterprise-level quality.

Reply:
Thanks Stetson for pointing this out. Feel free to suggest an improvement. I'd like to learn from this and apply it in the future.

Reply:
This is a style guide matter. I intentionally did not do style guide edits on the existing part of the doc. Because of timing, I suggest we merge first and I do this in the next PR.

Reply:
(For example, for a full style guide edit, we'd need to work out alternatives to "frontend" and "backend". And I would really want this readme to become available in the 7.5 download if at all possible)

Reply:
Another point after in-team discussion.
I would suggest that @emmurphy1 could be best positioned to do the style edit, as she is ramping up on OptaPlanner starting from the community side and has already done some community doc style/organizing work. Moreover she would probably improve on "my" section as well.